aggregate/housing_production/area_within_historic_district.py

```diff
@@ -31,7 +37,8 @@ def fraction_historic(geography_level):
     gdf[["area_historic_pct", "area_historic_sqmiles"]] = gdf.apply(
```

Comment:
my solution was just to change the second string in this line to `area_historic_sqmiles_count`